src/Html/Select.php

```diff
@@ -71,7 +71,18 @@ class Select
 	 */
 	public function booleanlist($name, $attribs = null, $selected = null, $yes = 'JYES', $no = 'JNO', $id = false)
 	{
-		$arr = array($this->option('0', $this->text->translate($no)), $this->option('1', $this->text->translate($yes)));
+		try
+		{
+			$optionNoText  = $this->getText()->translate($no);
+			$optionYesText = $this->getText()->translate($yes);
+		}
+		catch (\RuntimeException $exception)
+		{
+			$optionNoText  = $no;
```

Comment:
Should this not be hardcoded to english yes and no rather than use the lang strings again?